docs/resources/secrets_sync_flyio.md

```diff
@@ -0,0 +1,47 @@
+---
+page_title: "doppler_secrets_sync_flyio Resource - terraform-provider-doppler"
+subcategory: ""
+description: |-
+	Manage a Fly.io Doppler sync.
+---
+
+# doppler_secrets_sync_flyio (Resource)
+
+Manage a Fly.io Doppler sync.
+
+## Example Usage
+
+```terraform
+resource "doppler_integration_flyio" "prod" {
+  name    = "TF Fly.io"
+  api_key = "fo1_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+}
+
+resource "doppler_secrets_sync_flyio" "backend_prod" {
+  integration = doppler_integration_flyio.prod.id
+  project     = "backend"
+  config      = "prd"
+
+  app_id           = "my-app"
+  restart_machines = true
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `app_id` (String) The app ID
+- `config` (String) The name of the Doppler config
+- `integration` (String) The slug of the integration to use for this sync
+- `project` (String) The name of the Doppler project
+- `restart_machines` (Boolean) The Terraform Cloud variable set ID to sync to
+
+### Optional
+
+- `delete_behavior` (String) The behavior to be performed on the secrets in the sync target when this resource is deleted or recreated (leave_in_target by default)
```

Comment:
Thinking it might be good to list the valid options here (and the other places this is mentioned) since it's expecting a specific string.

Reply:
Great callout -- I assumed that would be automatically generated but I guess it makes sense that `ValidateFunc` isn't parsed